workAround.js

```diff
@@ -0,0 +1,144 @@
+let character = {//usado de exemplo
+	race : "human",//+1 skill qualquer
+	class : "bard",//+3 skills quaisquer
+	background : "criminal",//+deception e stealth
+	profBonus : 2,//personagem lvl 1 é sempre 2
+}
+
+
+//DEFINIÇÕES DE ATRIBUTOS
+let sheet = {
+  strength : 0,
+  dexterity : 0,
+  constitution : 0,
+  intelligence : 0,
+  wisdom : 0,
+  charisma : 0,
+}
+
+let standardArray = [15, 14, 13, 12, 10, 8];
+let atributes = ["strength", "dexterity", "constitution", "intelligence", "wisdom", "charisma"]
+let usedArray = [];
+let methodChoice=0;//se 0 usa standardArray, se 1 recebe input do usuario
+var atributoAuxiliar;
```

Comment:
This variable is written in pt-br. Change to es. Also is there any specific reason for using var here? Can be change to let?